packages/@aws-cdk/toolkit-lib/lib/toolkit/toolkit.ts

```diff
@@ -177,6 +180,15 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
             usePreviousParameters: parameters?.keepExistingParameters,
           },
         );
+
+        const result: EnvironmentBootstrapResult = {
+          environment,
+          status: bootstrapResult.noOp ? 'no-op' : 'success',
+          message: bootstrapResult.noOp ? 'No changes required' : 'Successfully bootstrapped',
```

Comment:
this seems like duplication of the status, do we need this?

Reply:
Agree that it presents the same information. We probably don't need message as we already have a human-readable message via the IOHost. Will remove it.